src/libexpr/primops/context.cc

```diff
@@ -119,7 +136,26 @@ static void prim_getContext(EvalState & state, const PosIdx pos, Value * * args,
     v.mkAttrs(attrs);
 }
 
-static RegisterPrimOp primop_getContext("__getContext", 1, prim_getContext);
+static RegisterPrimOp primop_getContext({
+    .name = "__getContext",
+    .args = {"s"},
+    .doc = R"(
+      Return the context associated with string *s*.
```

Comment:
It's too bad that the concept of "string context" is not defined anywhere in the manual, otherwise we could link to it here...